coordinator/tests/services/translation/request/agent-attended.spec.ts

```diff
@@ -0,0 +1,93 @@
+import {odsClient} from "../../../../src/services/communication/ods-client"
+import {createAuthorForAttendedAccess} from "../../../../src/services/translation/request/agent-attended"
+import pino from "pino"
+import {fhir, processingErrors as errors} from "@models"
+import {toArray} from "../../../../src/services/translation/common"
+
+const logger = pino()
+
+jest.mock("../../../../src/services/communication/ods-client", () => ({
+  odsClient: {
+    lookupOrganization: jest.fn()
+  }
+}))
+
+test("user details contain placeholder data - attended", async () => {
+  const mockLookupFunction = odsClient.lookupOrganization as jest.Mock
+  mockLookupFunction.mockReturnValueOnce(Promise.resolve(mockOrganizationResponse))
+
+  const result = await createAuthorForAttendedAccess("7654321", logger, "FTX40")
+
+  const agentPerson = result.AgentPerson
+  expect(agentPerson.id._attributes.extension).toEqual("999999999999")
+  expect(agentPerson.code._attributes.code).toEqual("R9999")
+  expect(agentPerson.telecom[0]._attributes.value).toEqual("tel:01234567890")
+
+  const agentPersonPerson = agentPerson.agentPerson
+  expect(agentPersonPerson.id._attributes.extension).toEqual("7654321")
+  expect(toArray(agentPersonPerson.name.given)[0]._text).toEqual("Attended")
+  expect(toArray(agentPersonPerson.name.family)[0]._text).toEqual("Access")
+})
```

Comment:
I tend to find unit tests are most useful and maintainable when we have one assertion per test

so here we could either try to condense these expects down to a single one (expect(agentPerson).toStrictEqual(...)) or we could split this test into a describe block i.e

`
describe("user details contain placeholder data - attended") {
    test("the agent person data is correct") {
        expects from lines 22-24 condensed to one assertion
    }
    test("the agent person person data is correct") {
        expects from lines 27-29 condensed to one assertion
    }
`

test names could be better 😅

Reply:
that describe block is looking good :+1: